MVCCore.Models/Login.cs

```diff
@@ -19,9 +19,6 @@ public class Login
         [Display(Name = "Password")]
         [MaxLength(50)]
```

Comment:
why there is data validation attribute

Reply:
Which one? only Display Name and max length is used

Reply:
max length validation attribute